pkg/users/accounts-list.js

```diff
@@ -109,7 +109,9 @@ const getGroupRow = (group, accounts, isUserCreatedGroup) => {
                 <Flex alignItems={{ default: 'alignItemsCenter' }}>
                     <div className={"dot " + groupColorClass} />
                     <FlexItem>
-                        <a href={"#/group/" + group.name}>{group.name}</a>
+                        <Button isInline variant="link" onClick={() => cockpit.jump("/users#/group/" + group.name)}>
```

Comment:
commit "users: try to replace href with buttons to fix missed onlocationchanged events": If you want to keep this, please update the commit message.

![do not try](https://www.livingwithbeth.com/wp-content/uploads/2015/05/Yoda-135x150.jpg)